db/migrations.go

```diff
@@ -1489,6 +1489,21 @@ var migrations = []func(*sql.Tx) error{
 				and val->>'rootURL' = 'http://localhost'`,
 		)
 	},
+	func(tx *sql.Tx) (err error) {
+		return execAll(tx,
+			`create table youtube_videos (
+				id text primary key,
+				title text not null,
+				thumb text not null,
+				video text not null,
```

Comment:
Use varchar with a reasonable size limit for both new tables. Letting an external API write gigantic strings into the DB is a security hazard.